packages/terra-consumer-nav/src/_spacing_utility.scss

```diff
@@ -0,0 +1,32 @@
+.u-mh--md {
```

Comment:
just make these styles inline in the nav.... we're moving away from utlities as generally when they are used they should jsut be props on the component. They also make theming difficult to apply